### PR TITLE
Development

### PR DESCRIPTION
**Pull request by codeDragon491**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+        const cardContainer = document.getElementById("cards-container");
+
+        cards.forEach((card) => {
+            let cardElement = document.createElement('div');
+            cardElement.classList.add("card");
+            cardElement.innerHTML = card;
+            cardContainer.appendChild(cardElement);
+        })
+
+        document.getElementById("shuffle-button").addEventListener("click", function() {
```
Comment (Tacsiazuma) [inline index.html]: If you created a local variable for the container, you could done it for these as well.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+            let oldCards = document.querySelectorAll(".card");
+            for (let i = 0; i < oldCards.length; i++) {
+                oldCards[i].innerHTML = shuffledCards[i];
+            }
+        })
+
+        document.getElementById("sort-button").addEventListener("click", function() {
+            let ascendingCards = cards.sort((a, b) => a - b);
+            let oldCards = document.querySelectorAll(".card");
+            for (let i = 0; i < oldCards.length; i++) {
```
Comment (Tacsiazuma) [inline index.html:207]: Can you see a pattern here and in the previous callback? Can we extract this logic somehow and reuse it in a function?

Comment (codeDragon491) [reply index.html:207]: yes, which solution do you like more? probably the first...

```
 shuffleButton.addEventListener("click", function() {
            swapCards(true)
        })

        sortButton.addEventListener("click", function() {
            swapCards()
        })

        function swapCards(swaped = false) {
          let swapedCards = cards.sort((a, b) => swaped ? 0.5 - Math.random() : a - b );
          let oldCards = document.querySelectorAll(".card");
            for (let i = 0; i < oldCards.length; i++) {
                oldCards[i].innerHTML = swapedCards[i];
            }
        }
```

or 
```
  shuffleButton.addEventListener("click", function() {
            let shuffledCards = cards.sort((a, b) => 0.5 - Math.random());
            swapCards(shuffledCards)
        })

        sortButton.addEventListener("click", function() {
            let ascendingCards = cards.sort((a, b) => a - b);
            swapCards(ascendingCards)
        })

        function swapCards(swapedCards) {
          let oldCards = document.querySelectorAll(".card");
            for (let i = 0; i < oldCards.length; i++) {
                oldCards[i].innerHTML = swapedCards[i];
            }
        }
```

Comment (Tacsiazuma) [reply index.html:207]: You should generally avoid boolean flag parameters. Even though it seems like a good idea in terms of code reuse its confusing. So I'd go with the second solution.

**Pull request merged**